examples/go.mod

```diff
@@ -9,3 +9,5 @@ require (
 )
 
 replace github.com/unidoc/unichart => ../
+
+replace github.com/unidoc/unipdf/v3 => ../../unipdf-src
```

Comment:
maybe not a good idea to put those local replacements into the repo?
You can create a go.work workspace file in your Github project folder and do those replacements there, that is a pretty good way to work when working with lib development.

Reply:
@gunnsth I think you are viewing an old commit here? Indeed I've mistakenly commit this replacement sometimes a go but I've removed it.

Reply:
Ah yes, seems so.  Thanks.